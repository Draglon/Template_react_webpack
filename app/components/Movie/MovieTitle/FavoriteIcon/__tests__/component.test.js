import React from 'react';

import FavoriteIconComponent from '../component';

describe('<FavoriteIconComponent />', () => {
  const props = {
    favorite: false,
    handleFavorite: () => {},
  };
  const component = shallow(<FavoriteIconComponent {...props} />);

  it('should match snapshot - favorite (false)', () => {
    expect(component).toMatchSnapshot();
  });

  it('should match snapshot - favorite (true)', () => {
    const newProps = {
      ...props,
      favorite: true,
    };
    component.setProps(newProps);
    expect(component).toMatchSnapshot();
  });
});
