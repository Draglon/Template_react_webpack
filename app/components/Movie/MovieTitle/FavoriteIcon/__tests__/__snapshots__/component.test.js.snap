// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoriteIconComponent /> should match snapshot - favorite (false) 1`] = `
<Icon
  onClick={[Function]}
  type="book"
/>
`;

exports[`<FavoriteIconComponent /> should match snapshot - favorite (true) 1`] = `
<Icon
  onClick={[Function]}
  theme="filled"
  type="book"
/>
`;
